sentences/nl/_common.yaml

```diff
@@ -311,13 +311,20 @@ expansion_rules:
   # generic expansion rules for sentences
   name: "[de|het] {name}"
   area: "[de|het] {area}"
+  in: "[in|op|van|bij]"
+  met: "(door|met|bij)"
+  name_area: >
+    (
+      [[door|met|bij] [de|het] {area}][ ]{name}
+      |[<in> [de|het] {area}] [door|met|bij] <name>
```

Comment:
doesn't it match now on the whitespace around `|` ?

Reply:
there is no `|` before the first one, and also no `|` after the third option. `<name>` is mandatory in all 3 options. So as far as I can see it doesn't match on whitespace only